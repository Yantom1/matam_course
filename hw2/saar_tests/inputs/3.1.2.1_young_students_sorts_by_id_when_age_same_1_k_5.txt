30000000, stu, 20, 2000, 1
10000000, stude, 20, 2000, 1
50000000, s, 20, 2000, 1
40000000, st, 20, 2000, 1
20000000, stud, 20, 2000, 1
