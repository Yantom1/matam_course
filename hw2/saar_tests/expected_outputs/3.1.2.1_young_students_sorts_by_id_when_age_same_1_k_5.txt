stude
stud
stu
st
s
